rust/hyperlane-base/src/settings/chains.rs

```diff
@@ -279,6 +279,27 @@ impl ChainSetup {
         .context("Building ValidatorAnnounce")
     }
 
+    /// Try to convert the chain setting into a Multisig Ism contract
+    pub async fn build_ism(&self, address: H256, metrics: &CoreMetrics) -> Result<Box<dyn Ism>> {
```

Comment:
Does the config not already have the ISM address? Feels odd since the others are all using `self.locator(&self.addresses.<name_of_contract>)`

Reply:
There are no enshrined ISM addresses, recipients specify whatever ISM address they like.

Reply:
Yes, but a given instance will still be attached to a single one, right?

Reply:
No, a given instance will not be attached to a single one, the relayer works with whatever ISM is specified by the recipient of a given message.